.gitignore

```diff
@@ -1,3 +1,7 @@
+# Rust
 /target
 **/*.rs.bk
 Cargo.lock
+
+# Editors
+.vscode
```

Comment:
You should put this in `.gitignore` in your home directory rather than in project gitignore configurations.